.travis/prepare.sh

```diff
@@ -1,14 +1,35 @@
 #!/bin/bash
 
-cd ../..
-mkdir cilium
-mv pchaigno/cilium cilium/cilium
-cd cilium/cilium
-
-curl https://apt.llvm.org/llvm-snapshot.gpg.key | sudo apt-key add -
-sudo apt-add-repository "deb http://apt.llvm.org/bionic/ llvm-toolchain-bionic-10 main"
-sudo apt-get -qq update
-sudo apt-get install -y clang-10 llvm-10
+CLANG_VERSION=10.0.0
+
+function setup_env() {
+case `uname -m` in
+  'x86_64' )
+    CLANG_DIR="clang+llvm-$CLANG_VERSION-x86_64-linux-gnu-ubuntu-18.04"
+    ;;
+  'aarch64' )
+    CLANG_DIR="clang+llvm-$CLANG_VERSION-aarch64-linux-gnu"
+    ;;
+esac
+}
+
+function install_clang() {
+  CLANG_FILE="$CLANG_DIR.tar.xz"
+  CLANG_URL="https://github.com/llvm/llvm-project/releases/download/llvmorg-$CLANG_VERSION/$CLANG_FILE"
+
+  wget -nv $CLANG_URL
+  sudo rm -rf /usr/local/clang
+  sudo mkdir -p /usr/local
+  sudo tar -C /usr/local -xJf $CLANG_FILE
+  sudo ln -s /usr/local/$CLANG_DIR /usr/local/clang
+  rm $CLANG_FILE
+}
+
+setup_env
+install_clang
+
+NEWPATH="/usr/local/clang/bin"
+export PATH="$NEWPATH:$PATH"
```

Comment:
Nit: these two lines could be combined as
```suggestion
export PATH="/usr/local/clang/bin:$PATH"
```

Reply:
Done, Please review.